Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -33,6 +33,11 @@ type Event = Object;
 var DEFAULT_PROPS = {
   activeOpacity: 0.85,
   underlayColor: 'black',
+  tvParallaxDisable: false,
+  tvParallaxShiftDistanceX: 2.0,
+  tvParallaxShiftDistanceY: 2.0,
+  tvParallaxTiltAngle: 0.05,
+  tvParallaxMagnification: 1.0
```

Comment:
Just committed changes to move all the Apple TV specific stuff to RCTTVView and RCTTVViewManager on the native side, and TVView on the JS side.